### DISCUSSION TITLE
Utilização do Grimoirelab para acompanhar a comunidade do Querido Diário

### DISCUSSION THREAD

**Discussion by jvanz** [okfn-brasil/querido-diario-comunidade#5] [Ideas]
Acredito que a comunidade do Querido Diário e a OKFN Brasil de maneira geral pode utilizar o projeto [Grimoirelab](https://chaoss.github.io/grimoirelab/) para acompanhar a comunidade. Esse projeto permite cruzar dados de várias fontes diferentes para avaliar como a comunidade do Querido Diário está se comportando e evoluindo.

**Comment by jvanz** [comment]
CC: @giuliocc @ArianeCamilo @campagnucci

**Comment by ogecece** [comment]
@jvanz será que a gente conseguiria conversar sobre esse projeto e talvez movimentar a comunidade pra puxar essa frente?

**Reply by jvanz** [reply]
Sim, me passa uns 3 dias que vocês podem conversar que eu escolho o que for melhor pra mim.

**Reply by jvanz** [reply]
@giuliocc :arrow_up:
